### ISSUE TITLE
Ternary operator should be placed directly below expression

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#518]
We should report an issue in case the ternary operator is not placed directly below the expression's position.

Following example should produce a violation:
```c#
var something = xyz is true
   ? 42
   : -1;
```

Following example should **NOT** produce a violation:
```c#
var something = xyz is true
                ? 42
                : -1;
```
